builder/amazon/common/access_config.go

```diff
@@ -86,3 +100,24 @@ func (c *AccessConfig) Prepare(t *packer.ConfigTemplate) []error {
 
 	return nil
 }
+
+func GetInstanceMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
Do you think this should be a constant? I'm surprised this isn't functionality exposed by the SDK. Can you check on that?